src/axolotl/utils/models.py

```diff
@@ -266,7 +292,7 @@ def load_llama_adapter(model, cfg):
         task_type="CAUSAL_LM",
     )
 
-    if cfg.peft_model_dir:
+    if cfg.lora_model_dir:
         model = PeftModel.from_pretrained(
```

Comment:
Add a log, so that we know it's being loaded
```suggestion
        logging.info("Loading pretained LORA")
        model = PeftModel.from_pretrained(
```